Update openhab_default.cfg

fazioa opened the pull request:
Configuration of Souliss Binding

buildhive commented:
[openHAB » openhab #3098](https://buildhive.cloudbees.com/job/openhab/job/openhab/3098/) FAILURE
Looks like there's a problem with this pull request
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

teichsta commented:
Thanks @fazioa!

fazioa commented:
Hi,
I have a question.
When I removed my thread and moved their function inside execute() the
binding have a problem.
When item files change, the binding update but execute stop to refresh.
How can I do to restart refreshing of "protected void execute()" ?




Tonino Fazio

2015-05-31 20:49 GMT+02:00 Thomas Eichstädt-Engelen <
notifications@github.com>:

> Thanks @fazioa <https://github.com/fazioa>!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openhab/openhab/pull/2667#issuecomment-107233251>.
>
